setup.py

```diff
@@ -107,7 +107,7 @@
   install_requires = ['six>=1.11.0', 'subprocess32; python_version < "3"'],
   extras_require = {
       'colors': ['colorama>=0.3.9'],
-      'crypto:python_version < "3"': ['cryptography>=3.3.2,<3.4'],
+      'crypto:python_version < "3"': ['cryptography>=3.3.2,<3.5'],
```

Comment:
this does not look right

Reply:
Nope, it's cool though that it didn't touch the python < 3 line in requirements-pinned.txt.

Reply:
So close!